node/src/ortc.ts

```diff
@@ -1109,43 +1109,60 @@ export function canConsume(
  *
  * It reduces encodings to just one and takes into account given RTP capabilities
  * to reduce codecs, codecs' RTCP feedback and header extensions, and also enables
- * or disabled RTX.
+ * or disables RTX.
  */
 export function getConsumerRtpParameters(
-	consumableParams: RtpParameters,
-	caps: RtpCapabilities,
-	pipe: boolean
+	{
+		consumableRtpParameters,
+		remoteRtpCapabilities,
+		pipe,
+		enableRtx
+	}:
+	{
+		consumableRtpParameters: RtpParameters;
+		remoteRtpCapabilities: RtpCapabilities;
+		pipe: boolean;
+		enableRtx: boolean;
+	}
 ): RtpParameters
 {
 	const consumerParams: RtpParameters =
 	{
 		codecs           : [],
 		headerExtensions : [],
 		encodings        : [],
-		rtcp             : consumableParams.rtcp
+		rtcp             : consumableRtpParameters.rtcp
 	};
 
-	for (const capCodec of caps.codecs!)
+	for (const capCodec of remoteRtpCapabilities.codecs!)
 	{
 		validateRtpCodecCapability(capCodec);
 	}
 
 	const consumableCodecs =
-		utils.clone(consumableParams.codecs) as RtpCodecParameters[];
+		utils.clone(consumableRtpParameters.codecs) as RtpCodecParameters[];
 
 	let rtxSupported = false;
 
 	for (const codec of consumableCodecs)
 	{
-		const matchedCapCodec = caps.codecs!
+		const matchedCapCodec = remoteRtpCapabilities.codecs!
 			.find((capCodec) => matchCodecs(capCodec, codec, { strict: true }));
 
 		if (!matchedCapCodec)
 		{
 			continue;
 		}
 
-		codec.rtcpFeedback = matchedCapCodec.rtcpFeedback;
+		if (!enableRtx && isRtxCodec(codec))
+		{
+			continue;
+		}
```

Comment:
Looks like cheaper operation that can go at the beginning of `for` loop. No need to iterate over codecs and match if this doesn't pass.